src/Connector.test.js

```diff
@@ -115,4 +116,24 @@ describe('Connector', () => {
     }
     expect(error).toBe(errors.registerListeners);
   });
+
+  test('clearListeners: should unregister listeners when there is no error while unsubscribing consumers', async () => {
```

Comment:
I'm not sure we have to test this operation since this is only a helper. Actually, it shouldn't even be exported (move out of the class).

Reply:
For now, let's continue testing it.

Reply:
Removed!